src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs

```diff
@@ -26,6 +27,7 @@ public DelugeSettings()
             Port = 8112;
             Password = "deluge";
             TvCategory = "tv-sonarr";
+            TvImportedCategory = "tv-sonarr";
```

Comment:
This default is going to be an issue for existing installs that aren't using the default. It's a forced behaviour change and Sonarr will start changing the category which isn't what I'd expect. We'll need a migration to avoid that.

Reply:
The default makes sense to me, but I don't know what goes into migrations so up to you what you want to do here.

Reply:
The migration would look like this: https://github.com/Sonarr/Sonarr/blob/phantom-develop/src/NzbDrone.Core/Datastore/Migration/033_add_api_key_to_pushover.cs

A separate class needs to be made for the JSON field to ensure a future migration doesn't break things and then some tests for validating the behaviour.

Reply:
I can try to make a skeleton or something, but I really won't be much use here. I don't know databases at all

Reply:
No, imo it should default to empty which should mean "don't change the category", which also means you don't need a migration.

Reply:
what if you just want to remove the label post import?

Reply:
Originally we wanted to disallow having no category in download clients, because it offers better segregation. But we couldn't for backward compat reasons.
iirc for some download clients Sonarr actually spits out a warning if you don't have a category.
So changing from a category to none after import seems odd, and something I rather not support.

The alternative is to have a checkbox as well to enable the ImportedCategory.

The reason is that you want to avoid that non-advanced users have to change two fields just to configure their normal category. So you need a way to have a default of 'don't change category on import'

Reply:
good point, I'll remove the default

Reply:
Should I change the help text to add something like "Leaving this empty means Sonarr won't change the category" ?

Reply:
Good idea. but maybe "Set to empty to disable changing the category" or something even shorter. Just go with what reads well on the UI :smile: